bookops_worldcat/query.py

```diff
@@ -53,8 +52,6 @@ def __init__(
         if session.authorization.is_expired():
             session._get_new_access_token()
 
-        self.response = None
```

Comment:
Not sure what I was thinking to have this line. Does not make sense now.
Thanks for catching it.